indico/modules/events/registration/util.py

```diff
@@ -577,3 +581,14 @@ def import_registrations_from_csv(regform, fileobj, skip_moderation=True, notify
         })
     return [create_registration(regform, data, notify_user=notify_users, skip_moderation=skip_moderation)
             for data in todo]
+
+
+def get_registered_contribution_participants(event_id):
```

Comment:
I'd pass the `event` itself as an argument even if you only needs the ID - that way it'd be easier to change things in the future if needed.